easybuild/tools/filetools.py

```diff
@@ -1166,12 +1167,15 @@ def rmtree2(path, n=3):
 
 def find_backup_name_candidate(src_file):
     """Returns a non-existing file to be used as destination for backup files"""
-    dst_file = src_file
 
-    i = 0
+    # e.g. 20170817234510 on Aug 17th 2017 at 23:45:10
+    timestamp = datetime.datetime.now()
+    dst_file = '%s_%s' % (src_file, timestamp.strftime('%Y%m%d%H%M%S'))
     while os.path.exists(dst_file):
```

Comment:
It does make sense, I wasn't trying to be safe w.r.t. having 2 EB instances running together, there's other scenarios:

* `back_up_file` being called twice on the same location in rapid succession; in theory this could happen when you run `--inject-checksums` twice in a row
* timezone changes making you 'go back' in time
* switch between summer/winter time resulting in going back 1 hour

So, I think it's worth having the check, it doesn't hurt anyway...